src/hooks/useStore.js

```diff
@@ -1,16 +1,20 @@
+import {nanoid} from 'nanoid';
 import create from 'zustand';
 
-const useStore = create(set => ({
-	counter: 0,
-	setCounter(counter) {
-		set({counter});
-	},
-	decrementCounter(step = 1) {
-		set(({counter}) => ({counter: counter - step}));
-	},
-	incrementCounter(step = 1) {
-		set(({counter}) => ({counter: counter + step}));
-	},
-}));
+const useStore = create(set => {
+	return {
+		items: [
+			{item: 'Zelt', id: nanoid(), isChecked: true},
+			{item: 'Grill', id: nanoid(), isChecked: false},
+			{item: 'Wein', id: nanoid(), isChecked: false},
+			{item: 'noch mehr Wein', id: nanoid(), isChecked: false},
```

Comment:
If I see correctly, right now you are not using the values saved under `isChecked`, therefore I would suggest to delete them.